unitTestSupportFunctions/getBodyNodes.m

```diff
@@ -0,0 +1,62 @@
+%% Function documentation
+%
+% Returns all non-boundary nodes that have boundary conditions set on them
+% Works only for 2D rectangular meshes 
+%
+%               Input :
+%              fldMsh : Nodes and elements for the fluid mesh
+%              homDBC : The global numbering of the DOFs where homogeneous
+%                       Dirichlet boundary conditions are applied
+%
+%              Output :
+%    nodalCoordinates : Nodes and elements from the fluid mesh that lie on
+%                       a body 
+%             homDOFs : The global numbering of the DOFs that lie on a body
+%                       where homogeneous Dirichlet boundary conditions are
+%                       applied
+%
+%% Function main body
+function homDOFs = getBodyNodes (fldMsh, homDBC)
```

Comment:
Is this function supposed to return all nodes on the boundary of a rectangular domain? If yes, change the name getBodyNodes to getBodyNodesOnRectangularDomain to make it more clear what it does